### ISSUE TITLE
Recommand to Remove Reddit r/bashonubuntuonwindows From the "endorsed community" page.

### ISSUE THREAD

**Issue by JaydenMcEncroe** [microsoft/WSL#7196]
**TLDR: Mods get mad of me correcting his false answer to a technical support post and permanently banned me for provide an accuate solution to other users.**

Seem a power tripping moderator in a technical support community about a technical detail is ... kind unprecedented.

I notice that 5 of the 10 moderators in that sub are Microsoft employees, either current of previous devolopers and PMs for the Windows Subsystem for Linux. I also notice that r/bashonubuntuonwindows is list under the [officially endorsed "Community" section](https://github.com/microsoft/WSL#community-links), may I suggest to make it clear what's Microsoft's position on this?

To summer up what happened, OP asked ["Can I use wslg without Windows 11?"](https://www.reddit.com/r/bashonubuntuonwindows/comments/omdzyv/can_i_use_wslg_without_windows_11/), 

> Hey! So, I wanted to upgrade my wsl to wslg, so I could do some SDL programming in wsl, but I noticed something.
>
> The github says that I have to join the dev channel in the insider program but the dev channel currently is "bloated" with Windows 11, which I don't really want to test right now.
>
> Is there any possibility to go around this and just get wslg?
>
> Another question I have in mind, will there be access to USB Devices from WSL too?
>
> Thanks

for some reason mods decided to lock that thread, [remove genuine answer by others](https://www.reddit.com/r/bashonubuntuonwindows/comments/omdzyv/can_i_use_wslg_without_windows_11/h5krghz) and [push his fake statement](https://www.reddit.com/r/bashonubuntuonwindows/comments/omdzyv/can_i_use_wslg_without_windows_11/h5kbcir), claiming

> I have locked the comments because of OPs unneeded derogatory comments, and because the answer I proved is clear. NO. Currently the developer insider build is required, and that is W11. The answer to the question is NO.

First, let forget about how using the word "bloated" to description Windows is "derogatory comments", but focouse on the answer he gives: "The answer to the question is NO" is clearly a false statement. [Microsoft still provides ISO Downloads Windows Insider Preview on their official websites](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewiso), and with the help of [open source UUP dumping tool](https://github.com/uup-dump), anyone can easily access tha latest Windows 10 Dev Builds including WSLg without the need of installing Windows 11.

Since OP's post was locked, I made another post titled ["Yes, You Absolutly Can Use WSLg Without Windows 11"](https://www.reddit.com/r/bashonubuntuonwindows/comments/omkdpd/yes_you_absolutly_can_use_wslg_without_windows_11/), just so that when people search the question on Google, they can get the accuate answer.

And the post get removed right after he saw it. [Here is the full exchange in that post.](https://i.loli.net/2021/07/19/V961epxk8jBbYFc.png)

At that same time, I saw another post seeking help for [Can't start WSL2 after changing ~/.profile](https://www.reddit.com/r/bashonubuntuonwindows/comments/omv33u/cant_start_wsl2_after_changing_profile/), the mod tell him to create another WSL distro and mount the original vhd on it. I corrected him and say access file directly under `\\wsl$\` path is way more easier.

I guess that is the final nail in the coffin, because right after that, I received a modmail telling me I was permanently banned, with the following reasoning:

> Being deeply, aggressively in civil with your comments. The number of strike an account gets is proportional to the quality input they give in the subreddit.
> 
> You have only appeared to be ... well deeply aggressively uncivilized. You may think that behavior is acceptable when directed to people who are a moderator. That would actually make it worse.
>
> Had you actively gone out of your way to provoke negative personal responses with any user here, or on reddit, you would have been banned.

The mods then creates an ["Official Announcement"](https://www.reddit.com/r/bashonubuntuonwindows/comments/omr6ht/tldr_wslwslg_windows_11_and_windows_10/) stating all discussing on the matter will be removed. He of course faces backlash from other members, who have no idea why such discussing will be an "issue". Rest assured, their replies and the whole post get removed (not sure if they get banned or not), and [he make another pinned post](https://www.reddit.com/r/bashonubuntuonwindows/comments/on2aoq/tldr_wslwslg_windows_11_and_windows_10/), this time totally locked to surpress and confusing and disagreement.

P.S. Their top moderator is also very active in political discussion and make comments related to various ...... to put it gently, right-wing ideas. I am not accusing him for any political position, but anyone look at his post history can make up their own mind.

**Comment by benhillis** [comment]
Thanks for the post, I will reach out to the mod.

**Comment by EricPell** [comment]
As the moderator of that sub reddit, I must say that what the use has said is a complete record of the exchange is inaccurate.

There were repeated, polite and directed requests over a period of time to stop using the WSL subreddit to complain or make inflammatory remarks about hardware requirements of W11. 

Despite these requests, a collection of uses took every opportunity to use posts reminding people that it was out of topic to post, in said posts, the same content. 

The subreddit has a HISTORY of staying away from unsupported claims, so that users can trust that the information they get is unbiased.

In response to this, the problematic users directly accused me, and another moderator of being MS employees, and biased.

When we both explained this was not the case, and said that we volunteer our time, we were told "we knew what we signed up for".

Third, I am politically active, and very much so against right wing extremist ideology. I believe this is independent of software, as being anti-fascist is ... I don't actually know how to say this, the factually correct position. 

I won't reply further to trouble makers who insist it is their right to make a mess because they want to.

For the record, I was not the moderator who muted the above user, it was an independent moderator who observed their abusive and unacceptable behavior. This post is a reply that I would make public an accounting of why their account was TEMPORARILY suspended, in keeping with my policy of informing the community why actions occurred. I expect this to be some type of damage control. Why? I don't know. 

As far as I can tell the repeated acceleration was viewed by multiple moderators and decided to be unacceptable. If stopping that behavior is unacceptable then there is a fundamental impasse that needs to be resolved.

**Comment by JaydenMcEncroe** [comment]
In response to @EricPell :

> In response to this, the problematic users directly accused me, and another moderator of being MS employees, and biased.
> 
> When we both explained this was not the case, and said that we volunteer our time, we were told "we knew what we signed up for".

There was never such conversation between me, and any moderators of such sub. My entire comment history are described as linked above with screenshots and anyone can verify it with removeddit.com. So why are you lying?

> This post is a reply that I would make public an accounting of why their account was TEMPORARILY suspended

Here is the modmail that inform me that I was permanently banned. Again, why are you lying?

> Third, I am politically active, and very much so against right wing extremist ideology. I believe this is independent of software, as being anti-fascist is ... I don't actually know how to say this, the factually correct position.

You have an active comment history in r/EnoughCommieSpam, r/fucktheccp, r/TumblrInAction and r/Conservative, I don't know anyone can call themself anti-fascist after paticipate in these sub.

**Comment by EricPell** [comment]
As I said, you and another participant were making a mess of things.

In the event you were not the other participant, you lack the context of knowing what was occuring that was along the same line.

" r/EnoughCommieSpam, r/fucktheccp, r/TumblrInAction and r/Conservative, "

I have never participated in those sub-reddits except for /r/Conservative. I suspect my account may be compromised, and I enabled 2 factor authentication some time ago.

**Comment by EricPell** [comment]
In regards to that account, which was one of many to be problematic on that day.

I said you were deeply, aggressively uncivil. That means you were intentionally being uncivil. You were actively doing what you were not asked to do. 

You repeated information to users that included unsupported builds against the direct rules of the subreddit, as informed by me. 

We don't have room for people who actively create problems.  You took an opportunity of those interactions to escelate your complaints and make personal statements. 

As I said in the ban message, for the record, the amount of changes is proportional to your participation. You had no participation history in the subreddit, and thus the odds you were creating issues for the sake of creating them out weighs the zero contributions you made.

Period.

**Comment by EricPell** [comment]
> Thanks for the post, I will reach out to the mod.

Ben, I will reply with a focused statement here.

A group of accounts repeatedly said that WSLg was coming to Windows 10 via update 21H2. The post they reference said that compute capability was coming. I flagged the statement that this was WSLg as inaccurate. 

The accounts then proceeded to repost that information every time I tried to clarify it, in posts or in comments. The nature of those comments was  a mixed bag, bouncing between it coming, and saying MS was arbitrarily withholding WSLg from W10 with all manor of negative statements. 

Had those claims been substantiated it would have been allowed to stand.   I made a clarifying post instructing people to "not complain about speculation" that we had other things to do. These same accounts used that post to REPOST links to deprecated builds, or to claim that W10 was getting WSLg. The same accounts which I had removed content asking to be changed or to refrain from doing these actions. 

The escalation lead to the ban. 

Please note, the above user has no history in the subreddit, except to highlight my accounts participation in extremists groups WHICH I DIDN'T EVEN KNOW ABOUT.

**Comment by JaydenMcEncroe** [comment]
In response to @EricPell :

> I said you were deeply, aggressively uncivil. That means you were intentionally being uncivil. You were actively doing what you were not asked to do.

Again, my comment history on that matter have already been fully disclosed in [the initial issue](https://github.com/microsoft/WSL/issues/7196#issue-947324716). You just can't make things up.

> You repeated information to users that included unsupported builds against the direct rules of the subreddit, as informed by me.

There are no such rules when I comment, you only change the sub rules AFTER you banned me.

> You had no participation history in the subreddit, and thus the odds you were creating issues for the sake of creating them out weighs the zero contributions you made.

[A quick google shows my comment history within the sub](https://i.loli.net/2021/07/20/faMlrpKBPiwWZ1U.png). As a mod, you have better tools to check my participation history, so why are you lying?

> A group of accounts repeatedly said that WSLg was coming to Windows 10 via update 21H2. The post they reference said that compute capability was coming. I flagged the statement that this was WSLg as inaccurate.
> 
> The accounts then proceeded to repost that information every time I tried to clarify it, in posts or in comments. The nature of those comments was a mixed bag, bouncing between it coming, and saying MS was arbitrarily withholding WSLg from W10 with all manor of negative statements.
> 
> Had those claims been substantiated it would have been allowed to stand. I made a clarifying post instructing people to "not complain about speculation" that we had other things to do. These same accounts used that post to REPOST links to deprecated builds, or to claim that W10 was getting WSLg. The same accounts which I had removed content asking to be changed or to refrain from doing these actions.

I never start, join, or paticipate any of the topic you mentioned above, anyone can check my post history and verify that I am telling the truth, so why are you trying to making things up?

> Please note, the above user has no history in the subreddit, no github history to speak of, except to highlight my accounts participation in extremists groups WHICH I DIDN'T EVEN KNOW ABOUT.

As I said, [a quick google shows my comment history within the sub](https://i.loli.net/2021/07/20/faMlrpKBPiwWZ1U.png). 

And [a look of your account history](https://www.reddit.com/user/Corrosive23/) also shows how you paticipate in various far-right sub (even including r/NoNewNormal. which is "a subreddit dedicated to spreading COVID-19 misinformation and disinformation", [according a Change.org petition](https://www.change.org/p/reddit-ban-r-nonewnormal-from-reddit)).

So why are you lying?

**Comment by EricPell** [comment]
That isn't my account.

So...

**Comment by JaydenMcEncroe** [comment]
> That isn't my account.
> 
> So... I mean, I explicitly am a distinct account from the other moderators. I created the subreddit.

I refered to "their top moderator" [in my initial issue](https://github.com/microsoft/WSL/issues/7196#issue-947324716), [This is the account of the top moderator in r/bashonubuntuonwindows](https://i.loli.net/2021/07/20/WL8g3vumt7O9TU5.png) as we speaking right now.

**Comment by EricPell** [comment]
"TOP moderator" could mean one of 100 things. Being at the top of a list I didn't care enough to structure was the least of my concern. Perhaps it means.... the one who does 99% of the moderating.  The "Top upvoted" moderator.  The one people actually interact with. It could mean one of a hundred things arbitrary at this time. 

Your ban is based on your behavior. The subreddit has a policy of absolutely zero tolerance regarding hate speech or anything approaching it, inside the subreddit or not. This will apply to moderators as well. 

However, as you DO have a history and I take that into account when brought to my attention, it will be temporary under the condition that you don't in the future take it upon yourself to override decisions of other moderators.

In either case, I am very grateful for the chance to investigate the issues you have raised, despite how it came to light.

**Comment by EricPell** [comment]
Also, I must insist you edit this posting immediately everywhere you intended to refer to an explicit account with explicit statements as to WHO YOU ARE REFERRING with regards to their social behavior

**Comment by JaydenMcEncroe** [comment]
> "TOP moderator" could mean one of 100 things.

In reddit, top moderator are defind strictly as "on the top of the moderator list", [as can be seen in a Reddit official post](https://www.reddit.com/r/modnews/comments/hr8hi6/an_update_regarding_top_moderator_permissions/).

**Comment by EricPell** [comment]
Are you intentionally continuing the combative interaction?

**Comment by benhillis** [comment]
I will continue this discussion offline, this is not an appropriate use of our GitHub.
